src/components/Favor/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as S from "./styled";
+
+interface User {
+  img: string;
+  name: string;
+  userName: string;
+}
+
+interface TFavor {
```

Comment:
Favor 컴포넌트의 Props를 정의한 타입이라면 아래와 같이 변경해보시는게 어떨까요?
```suggestion
interface TFavor { -> interface FavorProps
```
보통 컴포넌트의 Props를 정의한 타입은 타입의 뒤에 Props를 붙히곤 합니다.